src/actions/authentication.js

```diff
@@ -0,0 +1,48 @@
+import axios from 'axios';
+import jwt_decode from 'jwt-decode';
+import { GET_ERRORS, SET_CURRENT_USER } from './types';
+import setAuthToken from '../setAuthToken';
+
+
+export const registerUser = (user, history) => dispatch => {
```

Comment:
you can add REGISTER_PENDING, REGISTER_SUCESS and REGISTER_ERROR actions in order to control redux from redux dev tools and add some loading icon